### ISSUE TITLE
[Tool] Add a feedback message when the tool saves the grid model

### ISSUE THREAD

**Issue by FredericHirschmueller** [open-plan-tool/gui#27]
It should notify success or failure

**Comment by Bachibouzouk** [comment]
in #344 The save button is greyed out when the system is saved

**Comment by Bachibouzouk** [comment]
@stefansc1 - would it be possible for you to implement a small animation or automatically vanishing popup which states "the system was successfully saved" ? 
@bmlancien - did you already had a design for this?

**Comment by bmlancien** [comment]
> did you already had a design for this?

No, but I can do it.

**Comment by Bachibouzouk** [comment]
> > did you already had a design for this?
> 
> No, but I can do it.

Yes please :) 
@stefansc1 - you can wait for @bmlancien design

**Comment by bmlancien** [comment]
So I would actually think that the "save" button should be placed somewhere else, because "previous", "next" and "go to simulation" buttons are related, but not the "saved" button. You can see in the screenshot what I mean.

I think the feedback should be quite subtle, so I kept the toast just on top of the "save" button. If you think it should be more present, we could move it to the center of the page.

There is also a small [CodePen](https://codepen.io/michaelbr/pen/QwjEXjK) for showing the animation (styling is different than in the open_plan app, though, because I don't have the right CSS classes). A spin shows up in the button while saving, then a toast is shortly displayed, then removed, and the button goes back to disabled state.

<img width="1579" height="630" alt="Image" src="https://github.com/user-attachments/assets/f426d167-be00-47b1-90d9-b18dc3cca8af" />

**Comment by mstich0** [comment]
looking good this way, two thumbs up :D

**Comment by Bachibouzouk** [comment]
@stefansc1 or @josihoppe - can one of you tackle this with the codepen code linked by @bmlancien?

**Comment by josihoppe** [comment]
> [@stefansc1](https://github.com/stefansc1) or [@josihoppe](https://github.com/josihoppe) - can one of you tackle this with the codepen code linked by [@bmlancien](https://github.com/bmlancien)?

I can do this, I have some free capacity :)

**Comment by Bachibouzouk** [comment]
> > [@stefansc1](https://github.com/stefansc1) or [@josihoppe](https://github.com/josihoppe) - can one of you tackle this with the codepen code linked by [@bmlancien](https://github.com/bmlancien)?
> 
> I can do this, I have some free capacity :)

Then go  :)

**Comment by josihoppe** [comment]
Just saw, that on the pages step1 and step3 the simulation Button is on the right side, where the save button is now supposed to be from this issue. Should the other simulation buttons from the other steps also be moved to the middle, like the setup in step 2?

**Comment by Bachibouzouk** [comment]
> Should the other simulation buttons from the other steps also be moved to the middle, like the setup in step 2?

Yes if you can do this it would be great and re add the code to main.css
